src/workload.rs

```diff
@@ -117,10 +110,11 @@ pub mod address {
     #[derive(Debug, Eq, PartialEq, Clone, serde::Serialize, serde::Deserialize)]
     #[serde(untagged)]
     pub enum Address {
-        Workload(Box<Workload>),
-        Service(Box<Service>),
+        Workload(Workload),
+        Service(Service),
```

Comment:
does this pass cargo clippy / cargo fmt? i remember this triggering warnings about objects of very different size within an enum

Reply:
Ah, originally I had switched the entire interface to returning `Arc<...>`, which would also fix this. Since then I went the other way and have everything returning copies, so we need the `Box` again. Fixed.